### PR TITLE
Update multiple services example

### PR DESCRIPTION
**Pull request by ker0x**
This PR propose to replace SwiftMailer with the Mailer component and to add the Logger service for a better understanding how to inject multiple services

### CODE REVIEW

**Review by noniagriconomie — APPROVED** [review]

**Comment by OskarStark** [comment]
I like the idea but would not add the Logger in this example, yes it makes sense but complicated the example itself

**Comment by ker0x** [comment]
Hello @OskarStark , thank you for your feedback!

I have 3 propositions:

1. Removing the Logger from the example. This leave us with a try/catch that only return a boolean, it's a bit strange IMHO
2. Keeping the Logger and update the intro message of the [Handling Multiple Services](https://github.com/symfony/symfony-docs/blob/master/service_container.rst#handling-multiple-services) to specify that we want to log any exception that can be triggered when sending the email.
3. Replacing the Logger with an other component, but which one ?

**Comment by OskarStark** [comment]
Please use the same example as before, no need for a try catch here, it’s just an example and should not distract by things which are not needed.

Using

// ... 

Is fine too, meaning there could be some other code parts around. Thanks

File: service_container.rst [hunk 1]
```diff
@@ -311,18 +313,19 @@ made. To do that, you create a new class::
...
 
-            return $this->mailer->send($message) > 0;
+            $this->mailer->send($email);
+
+            // ...
         }
     }
 
-This needs the ``MessageGenerator`` *and* the ``Swift_Mailer`` service. That's no
+This needs the ``MessageGenerator`` *and* the ``Mailer`` service. That's no
```
Comment (HeahDude) [inline service_container.rst]: `MailerInterface`

**Review by HeahDude — COMMENTED** [review]
Looks good, thanks

File: service_container.rst (hunk 1 above)
Comment (ker0x) [reply service_container.rst]: Idk, I'm a little sceptical about this change. We call the `Mailer` service through the `MailerInterface` no?

Would't this sound better ?
> This needs the ``MessageGenerator`` *and* the ``Mailer`` service. That's no
problem, we ask them by type-hinting their class and interface names!

Comment (HeahDude) [reply service_container.rst]: Sounds better indeed

**Review by maxhelias — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Romain.

**Pull request merged**